### PR TITLE
nrf_wifi: Update RPU patch to rev#ff06137d3d7

### PR DESCRIPTION
**Pull request by karun2796**
UMAC:
         Updated the rpu path version to 1.2.10.5.
         [SHEL-2789] : Payload maximum value is always 70 in monitor mode.
 LMAC:
         [SHEL-2790] : Not able to filter for specific packet types
                       in monitor mode.

[SHEL-2789]: https://nordicsemi.atlassian.net/browse/SHEL-2789?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[SHEL-2790]: https://nordicsemi.atlassian.net/browse/SHEL-2790?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: nrf_wifi/fw_if/umac_if/inc/fw/host_rpu_sys_if.h [hunk 1]
```diff
@@ -1277,7 +1277,7 @@ struct nrf_wifi_event_raw_config_filter {
 	/** mode filter configured. */
 	unsigned char filter;
 	/** capture len configured. */
-	unsigned char capture_len;
+	unsigned int capture_len;
```
Comment (VivekUppunda) [inline nrf_wifi/fw_if/umac_if/inc/fw/host_rpu_sys_if.h:1280]: Can you please map it to unsigned short to map it to upper layer mapping. It is uint16 for driver and above

File: nrf_wifi/fw_if/umac_if/inc/fw/host_rpu_sys_if.h [hunk 2]
```diff
@@ -1174,7 +1174,7 @@ struct nrf_wifi_cmd_raw_config_filter {
 	/** Wireless device operating mode filters for Promiscuous/Monitor modes. */
 	unsigned char filter;
 	/** capture length. */
-	unsigned char capture_len;
+	unsigned int capture_len;
```
Comment (VivekUppunda) [inline nrf_wifi/fw_if/umac_if/inc/fw/host_rpu_sys_if.h:1177]: can you please map it to unsigned short to map it to upper layer mapping

File: nrf_wifi/fw_if/umac_if/inc/fw/host_rpu_sys_if.h (hunk 1 above)
Comment (karun2796) [reply nrf_wifi/fw_if/umac_if/inc/fw/host_rpu_sys_if.h]: Done

File: nrf_wifi/fw_if/umac_if/inc/fw/host_rpu_sys_if.h (hunk 2 above)
Comment (karun2796) [reply nrf_wifi/fw_if/umac_if/inc/fw/host_rpu_sys_if.h]: Taken care

**Review by VivekUppunda — APPROVED** [review]

**Review by sachinthegreen — CHANGES_REQUESTED** [review]
Please tag the relevant Jira reference in the commit message.

**Comment by karun2796** [comment]
> Please tag the relevant Jira reference in the commit message.

Updated.

**Review by sachinthegreen — APPROVED** [review]

**Pull request merged**